sonic_platform_base/sonic_storage/storage_common.py

```diff
@@ -0,0 +1,78 @@
+#
+# storage_common.py
+#
+# Class for common implementation functions,
+# i.e. functions that are storage device type agnostic.
+#
+
+try:
+    import os
+    import sys
+    import json
+except ImportError as e:
+    raise ImportError (str(e) + "- required module not found")
+
+try:
+    import psutil
+except ImportError as e:
+    pass
+
+
+class StorageCommon(object):
+    def __init__(self, diskdev):
+        """
+        Constructor
+
+        Args:
+            Block device path for which we need to get information
+        """
+        self.DISKSTATS_FILE = "/proc/diskstats"
+        self.diskdev = os.path.basename(diskdev)
+        self.fsstats_reads = 0
+        self.fsstats_writes = 0
+
+    def get_fs_io_reads(self):
+        """
+        Function to get the latest reads on the disk by parsing the /proc/diskstats file
+
+        Returns:
+            The total number of procfs reads
+
+        Args:
+            N/A
+        """
+
+        if 'psutil' in sys.modules:
```

Comment:
@assrinivasan can we use psutil only as it has standard implementation

Reply:
Will do.

Reply:
Done in latest commit